spec/feature/posts/post_show_spec.rb

```diff
@@ -0,0 +1,67 @@
+require 'rails_helper'
+
+RSpec.describe 'Post show page', type: :feature do
+  before do
+    @user = User.create(
+      name: 'Tom',
+      photo: 'https://unsplash.com/photos/F_-0BxGuVvo',
+      bio: 'Teacher from Mexico.',
+      posts_counter: 1
+    )
+
+    @post = Post.create(
+      title: 'Hello',
+      text: 'This is my first post',
+      comments_counter: 3,
+      likes_counter: 4,
+      user_id: @user.id
+    )
+
+    @first_comment = Comment.create(
+      post_id: @post.id,
+      user_id: @user.id,
+      text: 'Yeah it is'
+    )
+
+    @like = Like.create(user_id: @user.id, post_id: @post.id)
+    visit user_post_path(@user, @post)
+  end
+
+  it "displays the post's title" do
+    visit user_post_path(@user, @post)
+    expect(page).to have_content(@post.title)
+  end
+
+  it "displays the post's author" do
+    visit user_post_path(@user, @post)
+    expect(page).to have_content(@post.user_id)
+  end
```

Comment:
- [ ] Please make sure that all tests are passing.

![image](https://user-images.githubusercontent.com/62290545/231225320-52e12e2c-f43d-4227-a05e-101f4f17dbe4.png)